src/components/Dust/DustState.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
+
+interface DustStateProps {
+  dustDensity: string;
+  kindOfDust: string;
+}
+
+const dustRateColor = ['#1E64EE', '#00D500', '#F95A20', '#E73532'];
+const dustRate = ['좋음', '보통', '나쁨', '매우 나쁨'];
+const averageDustDensity = [2, 3, 4, 10];
+const fineDustDensity = [80, 150, 300, 1200];
+const ultraFineDustDensity = [15, 35, 75, 1200];
+
+const KIND_OF_DUST_AVG = 'avg';
+const KIND_OF_DUST_FINEDUST = 'fineDust';
+const KIND_OF_DUST_ULTRAFINEDUST = 'ultraFineDust';
```

Comment:
같은 관심사인데 하나로 묶는건 어떤가요? 다음과같이

```suggestion
const DUST_KIND = {
  AVG: 'avg',
  FINE_DUST: 'fineDust',
  ULTRA_FINE_DUST: 'ultraFineDust',
};
```